docs/documentation/running_a_simulation/Input-File.md

```diff
@@ -148,9 +148,11 @@ Structure and class variables do not carry around units, and therefore the units
 ## Creating New Objects and Allocating Memory
 
 It is possible to create new objects and allocate new memory for structures directly in the Python
-input file.  There are at least two ways to allocate memory.
+input file.  Three different ways are described below.
 
-The first method is to call the `Trick::MemoryManager` routines to allocate memory.  This is the preferred method.
+### 1. Call `Trick::MemoryManager` allocation routines directly.
```

Comment:
How about changing it to "Call Trick::MemoryManger Allocation Routines Directly" -- no period and capitalize the first letter of each word to be consistent

Reply:
Done.